solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -791,14 +791,13 @@ public JettySolrRunner getReplicaJetty(Replica replica) {
 
   /** Make the zookeeper session on a particular jetty expire */
   public void expireZkSession(JettySolrRunner jetty) {
+    ChaosMonkey.causeConnectionLoss(jetty);
+
```

Comment:
Am curious about (not) keeping the connection loss/close within the `cores != null` conditional before the session expiry.

Reply:
addressed in b386e144e2f